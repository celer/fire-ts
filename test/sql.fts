<%
	var colors=[ "red","blue","green"];
	var numbers=[1,2,3,5];
%>


-- %{schema} 

create table colors (
	id bigint auto_increment,
	color varchar(20) not null,
	number bigint not null,
);

-- }%

<% colors.map(function(color){ %>
	<% numbers.map(function(number){ %>
		insert into colors (name) values(<%#color%>,<%=number%>);
	<% }); %>
<% }); %>

