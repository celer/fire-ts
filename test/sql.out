

-- %{schema} 

create table colors (
	id bigint auto_increment,
	color varchar(20) not null,
	number bigint not null,
);

-- }%

insert into colors (name) values("red",1);
insert into colors (name) values("red",2);
insert into colors (name) values("red",3);
insert into colors (name) values("red",5);
insert into colors (name) values("blue",1);
insert into colors (name) values("blue",2);
insert into colors (name) values("blue",3);
insert into colors (name) values("blue",5);
insert into colors (name) values("green",1);
insert into colors (name) values("green",2);
insert into colors (name) values("green",3);
insert into colors (name) values("green",5);

